primary/src/async_proposer.rs

```diff
@@ -13,11 +13,11 @@ use tracing::info;
 use types::{BatchDigest, Certificate, CertificateDigest, Header, Round};
 
 #[cfg(test)]
-#[path = "tests/proposer_tests.rs"]
-pub mod proposer_tests;
+#[path = "tests/async_proposer_tests.rs"]
+pub mod async_proposer_tests;
```

Comment:
is proposer the best name? Maybe DAG_builder or block_builder?

Reply:
nope, I agree that dag_builder is better. We may defer this renaming for later though because most of our technical documentation uses the term "proposer"

Reply:
making an issue